http.py

```diff
@@ -114,8 +115,9 @@ def __init__(self, http_request):
         self.response_object = None
         self.request = None
         self.sock = None
-        self.CIPHERS = 'ADH-AES256-SHA:ECDHE-ECDSA-AES128-GCM-SHA256: \
-                ECDHE-RSA-AES128-GCM-SHA256:AES128-GCM-SHA256:AES128-SHA256:HIGH:'
+        self.CIPHERS = \
```

Comment:
Are these two strings equivalent, aka the old one and the new one? Can you run a quick test in shell and confirm? We may get undefined behavior if spaces are mismatched